spec/imsc-hrm.html

```diff
@@ -230,7 +232,7 @@ <h2>Terms and Definitions</h2>
     </p>
     <p>
       A sequence of consecutive <dfn data-cite="ttml2#terms-intermediate-synchronic-document">Intermediate Synchronic Documents</dfn>
-      conforms to the Hypothetical Render Model if is processed without error as defined in <a
+      conforms to the Hypothetical Render Model if is processed without <a>error</a> as defined in <a
```

Comment:
```suggestion
      conforms to the Hypothetical Render Model if is processed without <a>error</a>.
```

We cannot say _error_ is defined below since it is now defined in _terms and definitions_.